wordpress/content.md

```diff
@@ -36,3 +37,8 @@ container's IP, standard port mappings can be used:
 
 Then, access it via `http://localhost:8080` or `http://host-ip:8080` in a
 browser.
+
+If you'd like to use an external database instead of a linked `mysql` container
+specify the hostname and port with `WORDPRESS_DB_HOST`:
+
+    docker run --name some-wordpress -e WORDPRESS_DB_HOST=10.1.2.3:3306 -d wordpress
```

Comment:
(Possibly even `WORDPRESS_DB_USER`, but they should be smart enough to know to specify when the sql user is not `root`)